nextcloudappstore/scaffolding/forms.py

```diff
@@ -86,6 +87,28 @@ class IntegrationScaffoldingForm(Form):
                                         ' integration '
                                         'does. Can contain Markdown.'))
 
+    def _create_discourse_category(self, app_id: str) -> None:
+        url = '%s/categories?api_key=%s&api_username=%s' % (
+            settings.DISCOURSE_URL.rstrip('/'),
+            settings.DISCOURSE_TOKEN,
```

Comment:
As of now token and user are `None` ... let me set some up there first :)

Reply:
Ok. This was used in another place as well...

Reply:
Most likely on an non-authenticated API I think. Do you have a link?

Reply:
Found it ... Anyways - I created one.

Reply:
:)